content/en/docs/Responses/PlayQueue.md

```diff
@@ -8,12 +8,86 @@ description: >
 {{< tabpane persistLang=false >}}
 {{< tab header="**Example**:" disabled=true />}}
 {{< tab header="OpenSubsonic" lang="json">}}
-// TODO
+{
+  "current": "1234",
+  "position": 1000,
+  "username": "user",
+  "changed": "2023-03-10T02:19:35.784818075Z",
+  "changedBy": "example client",
+  "entry": [
+    {
+      "id": "1234",
+      "parent": "200000021",
+      "title": "Can I Help U?",
+      "isDir": false,
+      "isVideo": false,
+      "type": "music",
+      "albumId": "200000021",
+      "album": "Forget and Remember",
+      "artistId": "100000036",
+      "artist": "Comfort Fit",
+      "coverArt": "1234",
+      "duration": 103,
+      "bitRate": 216,
+      "bitDepth": 16,
+      "samplingRate": 44100,
+      "channelCount": 2,
+      "track": 1,
+      "year": 2005,
+      "genre": "Hip-Hop",
+      "size": 2811819,
+      "discNumber": 1,
+      "suffix": "mp3",
+      "contentType": "audio/mpeg",
+      "path": "user/Comfort Fit/Forget And Remember/1 - Can I Help U?.mp3"
+    }
+  ]
+}
 {{< /tab >}}
 {{< tab header="Subsonic" lang="json" >}}
-// TODO
+{
+  "current": "1234",
+  "position": 1000,
+  "username": "user",
+  "changed": "2023-03-10T02:19:35.784818075Z",
+  "changedBy": "example client",
+  "entry": [
+    {
+      "id": "1234",
+      "parent": "200000021",
+      "title": "Can I Help U?",
+      "isDir": false,
+      "isVideo": false,
+      "type": "music",
+      "albumId": "200000021",
+      "album": "Forget and Remember",
+      "artistId": "100000036",
+      "artist": "Comfort Fit",
+      "coverArt": "1234",
+      "duration": 103,
+      "bitRate": 216,
+      "bitDepth": 16,
+      "samplingRate": 44100,
+      "channelCount": 2,
+      "track": 1,
+      "year": 2005,
+      "genre": "Hip-Hop",
+      "size": 2811819,
+      "discNumber": 1,
+      "suffix": "mp3",
+      "contentType": "audio/mpeg",
+      "path": "user/Comfort Fit/Forget And Remember/1 - Can I Help U?.mp3"
+    }
+  ]
+}
 {{< /tab >}}
 {{< /tabpane >}}
 
-| Field |  Type | Req. | OpenS. | Details |
-| --- | --- | --- | --- | --- |
+| Field       | Type                         | Req.    | OpenS. | Details                                             |
+| ----------- | ---------------------------- | ------- | ------ | --------------------------------------------------- |
+| `current`   | `int`                        | No      |        | ID of currently playing track                       |
```

Comment:
`current` should be a `string` for the ID

Reply:
This might be an original spec issue, but the `current` is defined as an int?
```xml
<xs:attribute name="current" type="xs:int" use="optional"/>
```

Reply:
Yes, it is an issue with old spec. As Navidrome uses ID strings, I implemented this as string and it didn't break any OG clients.

Reply:
It seems that the OG Subsonic server actually does use an integer there so the spec is in line with the implementation: http://demo.subsonic.org/rest/getPlayQueue.view?c=test&v=1.16.1&f=json&u=guest&p=guest

But as that doesn't really make sense, should we still change it?

Reply:
Wow. 
From that, it looks like internally all of the IDs are internally an int (included in a string), so it technically works. Given that it's inconsistent with the spec, I'd argue saying it should be a string is more correct? (I added an errata for now)